react-viewer/tsconfig.json

```diff
@@ -13,8 +17,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Is this change needed?

Reply:
default change automatically made by cra when you use react 17; should be negligible

Reply:
okay, didn't know we changed that in the core. Fine by me